### DISCUSSION TITLE
what other table to truncate besides 'webcal_entry' to remove all entries from the calendar?

### DISCUSSION THREAD

**Discussion by marcnyc2001** [craigk5n/webcalendar#464] [General]
Hello I am trying to create a duplicate of my calendar and start from scratch but keep ALL settings exactly the same, except for the entries.
I tried truncating all entries in the table 'webcal_entry' but they still show on the calendar.
What else do I remove in MySQL to start without any events in the calendar?
Thank you!

**Comment by bbannon** [comment]
Either you didn't truncate the active table,
or you didn't commit the changes, (I can't count the number of times I did that)
or you or your ISP or someone have done a  'restore from backup' .
Did you actually run 'truncate' or 'delete FROM webcal_entry' or REMOVE and CREATE the table fresh?

**Comment by marcnyc2001** [comment]
I am not sure how to Commit. I didn't know I had to commit. Where/how would I do that?
I googled what you said and the MySQL manual says that Truncate operations cause an implicit commit (https://dev.mysql.com/doc/refman/8.0/en/truncate-table.html).

What I did was: in PhpMyAdmin I went to Operations and at the bottom I chose Empty the table (TRUNCATE)

now when I go to the webcal_entry table I get this:

MySQL returned an empty result set (i.e. zero rows). (Query took 0.0002 seconds.)
SELECT * FROM `webcal_entry`

But I keep seeing the data in my browser (even after refreshing and trying a private browser to make sure it wasn't a caching issue)

The strange thing is that if I click on any of the entries that I see, I get this:
Error
Invalid entry id: "xxxxxx".

so it's almost as if the entries don't exist but they do show up in the calendar

**Reply by Ganzfix** [reply]
Have you tried to empty the tmp directory?

**Reply by marcnyc2001** [reply]
No. Where is that?

**Reply by Ganzfix** [reply]
The path is in the file include/settings.php
db_cachedir: *tmp

**Reply by marcnyc2001** [reply]
I tried removing the /tmp directory on the server but nothing changes...
